tests/core/adjust/main.fmf

```diff
@@ -1,4 +1,4 @@
 summary: Verify test/plan adjustments based on context
 description:
     Perform a basic check using 'tmt test/plan show'.
-require+: [tmt+provision-container]
+#require+: [tmt+provision-container]
```

Comment:
Is this change really relevant to this PR?

Reply:
Nope, fixed in e2d256f8